src/Core/Components/InputFile/FluentInputFile.razor

```diff
@@ -2,19 +2,12 @@
 @using Microsoft.AspNetCore.Components.Forms
 @inherits FluentComponentBase
 
-<div class="@ClassValue"
+<div class="@ClassValue" 
      style="@StyleValue"
-     drop-files="@DropOver"
      disabled="@Disabled"
-     @ondragenter="@(e => DropOver = true)"
-     @ondragenter:stopPropagation
-     @ondragover="@(e => DropOver = true)"
-     @ondragover:stopPropagation
-     @ondragleave="@(e => DropOver = false)"
-     @ondragleave:stopPropagation
-     @ondragend="@(e => DropOver = false)">
+     @ref="@_containerElement">
 
-    <div class="inputfile-content" style="z-index: @(DropOver ? null : 999)">
+    <div class="inputfile-content">
```

Comment:
To verify but this `z-index` is important to have the possibility to click on panel links.